index.html

```diff
@@ -144,14 +147,14 @@ <h3 class="section-title">Our Global Partners</h3>
             <div><img class="logo-img" src="assets/images/logo/outlier.svg" alt="outlier-logo"></div>
             <div><img class="logo-img" src="assets/images/logo/standford.svg" alt="standford-logo"></div>
             <div><img class="logo-img" src="assets/images/logo/vmware.svg" alt="vmware-logo"></div>
-            <div><img class="logo-img" src="assets/images/logo/a[[teguu.svg" alt="apptegu-logo"></div>
+            <div><img class="logo-img" src="assets/images/logo/appteguu.svg" alt="apptegu-logo"></div>
         </div>
-      </div>
-      <!-- footer section -->
-      <div class="footer-section">
-        <p class="footer-text">&#169;2022 Micronauts Summit. No Right Reserverd | Profsain Hussein</p>
-      </div>
+      </section>
     </main>
+     <!-- footer section -->
+     <footer class="footer-section">
+      <p class="footer-text">&#169;2022 Micronauts Summit. No Right Reserverd | Profsain Hussein</p>
+    </footer>
```

Comment:
- [ ] Please add the logo to your footer, and sorry since it was not mentioned in previous reviews. here is the link to the Design. ([See original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015))
-
![miss2](https://user-images.githubusercontent.com/63972670/169516579-9e2a0074-2942-4ccf-941c-32dd66df784c.PNG)